js/libs/core.js

```diff
@@ -182,8 +180,20 @@ Fliplet.FormBuilder = (function() {
         type: Boolean,
         default: false
       };
+  
+      component.props._isCustomizeName = {
+        type: Boolean,
+        default: false
+      };
 
       component.computed._fieldNameError = function() {
+        
+        if (!this._isCustomizeName && !this.label) {
```

Comment:
`hasCustomName`, no need for the underscore prefix

Reply:
Don't quite understand what's wrong with that?
Need instead of _isCustomizeName to use hasCustomName and without underscore prefix?

Reply:
yes, rename `_isCustomizeName` to `hasCustomName`

Reply:
Fixed